src/screens/VisualTests/StoryInfo.tsx

```diff
@@ -73,19 +76,17 @@ export const StoryInfo = ({
         </small>
       </Text>
     );
-  } else if (isOutdated) {
+  } else if (isStoryOutdated && switchToNextBuild) {
```

Comment:
Looks like we're changing the definition of 'outdated' from "there are code changes, you need a new build" to "there's a new build, you need to apply it". I'd rather use different terminology, e.g. "stale".

Reply:
Superceded?